### PR TITLE
Active_trail_nodes accepts iterables

### PR DESCRIPTION
**Pull request by raghavg7796**
Solving #722 first.

### CODE REVIEW

File: pgmpy/models/BayesianModel.py [hunk 1]
```diff
@@ -410,9 +412,9 @@ def active_trail_nodes(self, start, observed=None):
         >>> student.add_nodes_from(['diff', 'intel', 'grades'])
         >>> student.add_edges_from([('diff', 'grades'), ('intel', 'grades')])
         >>> student.active_trail_nodes('diff')
```
Comment (khalibartan) [inline pgmpy/models/BayesianModel.py:414]: @raghavg7796 I'm not in a agreement with allowing strings for single variable. @ankurankan  I think we are doing this in other places also. Was there any particular reason for this ?

Comment (raghavg7796) [reply pgmpy/models/BayesianModel.py:414]: @khalibartan Bayesian Model specifies nodes should be strings so I was using strings for single variable.
@ankurankan But also there is another problem that add_edge method specifies its argument to be any hashable python objects. This is in contradiction with docstring of Bayesian Model.

**Comment by raghavg7796** [comment]
@ankurankan I think this is ready for merge. Please review.

File: pgmpy/models/BayesianModel.py (hunk 1 above)
Comment (ankurankan) [reply pgmpy/models/BayesianModel.py:414]: @raghavg7796 Earlier we used to support only strings for node names but later we refactored the code to accept any hashable object. So, we might have missed changing the docstring somewhere. Can you create an issue for that ?

And I think you will need to do some fixes for that in this PR. Right ?

Comment (raghavg7796) [reply pgmpy/models/BayesianModel.py:414]: @ankurankan OK! I will create the issue and make the required changes in this PR.

**Comment by ankurankan** [comment]
@raghavg7796 Let me know when this is ready.

**Comment by raghavg7796** [comment]
@ankurankan This is ready now.

**Pull request merged**